src/iomei.cpp

```diff
@@ -182,9 +183,27 @@ MEIOutput::MEIOutput(Doc *doc) : Output(doc)
 
 MEIOutput::~MEIOutput() {}
 
-bool MEIOutput::Export()
+bool MEIOutput::Skip(Object *object)
 {
+    if (object->Is(MDIV)) {
+        VisibilityDrawingInterface *interface = object->GetVisibilityDrawingInterface();
+        assert(interface);
+        if (!interface->IsHidden() || this->IsSerializing()) return false;
+        if (this->IsPageBasedMEI() || this->HasFilter()) return true;
+    }
+    else if (object->IsEditorialElement()) {
+        VisibilityDrawingInterface *interface = object->GetVisibilityDrawingInterface();
+        assert(interface);
+        if (!interface->IsHidden() || this - IsSerializing()) return false;
```

Comment:
Interesting error which actually compiles.
```cpp
this->IsSerializing()
```

Reply:
Yes, this is actually quite a nasty one. I [had](https://github.com/rism-digital/verovio/pull/4047/commits/7a10c416ee6758f9838acf50b9b508df154cd079) fix one but mist this one. Thanks!